flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobPathParameter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.rest.messages;
+
+import org.apache.flink.api.common.JobID;
+
+/**
+ * Path parameter identifying jobs.
+ *
+ * <p>The type of this parameter is {@link JobID}.
+ */
+public class JobPathParameter extends MessagePathParameter<JobID> {
```

Comment:
I was actually going to create the same class for https://issues.apache.org/jira/browse/FLINK-7694, as described in that ticket :)

I prefer `JobIDPathParameter`

Reply:
Alright. Will rename it.